### PR TITLE
Supported pagination for raw cosv files

### LINKED ISSUE #2679: Support pagination in raw cosv files

### PR DESCRIPTION
**Pull request by nulls**
### What's done:
- supported listing of raw cosv files by pages
- added a button to show more

It closes #2679 

raw-cosv-files-pagination

### CODE REVIEW

File: save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/SpringUtils.kt [hunk 1]
```diff
@@ -70,6 +74,15 @@ inline fun <reified T : BaseEntity, R : BaseEntityRepository<T>> R.getByIdOrNotF
...
 }
 
+/**
+ * Returns a [Page] of entities matching the given [Specification].
+ *
+ * @param spec can be null.
+ * @param pageable must not be null.
+ * @return never null.
+ */
+fun <T : Any> JpaSpecificationExecutor<T>.kFindAll(pageable: Pageable, spec: Specification<T>?): Page<T> = findAll(spec, pageable)
```
Comment (Cheshiriks) [inline save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/SpringUtils.kt:84]: Is this not used anywhere?

Comment (nulls) [reply save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/SpringUtils.kt:84]: Nope, but I don't want to lose it. Can be useful for future.
Will try to apply for existing code

Comment (nulls) [reply save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/SpringUtils.kt:84]: applied in another places: now they look more kotlinness

**Comment by nulls** [comment]
raw-cosv-files
raw-cosv-files-2
raw-cosv-files-3

**Pull request merged**